gaia-dialog-alert.js

```diff
@@ -17,6 +17,8 @@ module.exports = component.register('gaia-dialog-alert', {
     this.els = {
       dialog: this.shadowRoot.querySelector('gaia-dialog')
     };
+    this.els.dialog.addEventListener('closed',
+                                     GaiaDialogProto.hide.bind(this));
```

Comment:
NIT: Don't like this kind of alignment. Can we stick to 2 space all the time? :)

```js
this.els.dialog.addEventListener('closed',
  GaiaDialogProto.hide.bind(this));
```